lib/std/http/Server.zig

```diff
@@ -303,6 +303,51 @@ pub const Request = struct {
         return http.HeaderIterator.init(r.server.read_buffer[0..r.head_end]);
     }
 
+    /// Retrieves the value of a specified HTTP header from a Request object.
+    ///
+    /// This function searches for a header with a name matching the provided string,
+    /// ignoring case. If found, it returns the header's value. If not found, it
+    /// returns an empty string.
+    pub fn getHeader(r: *Request, s: []const u8) []const u8 {
```

Comment:
Please fix the const-correctness of `iterateHeaders()` instead of spreading it further. `getHeader()` should not need a mutable `Request` pointer.

Reply:
Fixed - thanks for the feedback.